src/geo/cartodb-layer-group.js

```diff
@@ -46,15 +46,19 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     return !!this.get('urls');
   },
 
-  getTileURLTemplates: function () {
+  getTileURLTemplates: function (type) {
+    type = type || 'png';
```

Comment:
There is no better way to check for type of tile than the extension? maybe it'll be useful to have some constant in case we have more extensions or whatever